### PR TITLE
bump version - 16.04.1

### PR DESCRIPTION
**Pull request by oinume**
Update to 16.04.1

### CODE REVIEW

File: ubuntu1604-desktop.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
   "desktop": "true",
   "cpus": "1",
   "disk_size": "130048",
-  "iso_checksum": "70db69379816b91eb01559212ae474a36ecec9ef",
+  "iso_checksum": "de5ee8665048f009577763efbf4a6f0558833e59",
   "iso_checksum_type": "sha1",
-  "iso_name": "ubuntu-16.04-server-amd64.iso",
-  "iso_url": "http://releases.ubuntu.com/16.04/ubuntu-16.04-server-amd64.iso",
```
Comment (oinume) [inline ubuntu1604-desktop.json]: I think this should be `ubuntu-16.04.1-desktop-amd64.iso`, right?

File: ubuntu1604-desktop.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
   "desktop": "true",
   "cpus": "1",
   "disk_size": "130048",
-  "iso_checksum": "70db69379816b91eb01559212ae474a36ecec9ef",
+  "iso_checksum": "805337c2c3a00ac9b4a59a5c9692903ad30fe3ce",
   "iso_checksum_type": "sha1",
-  "iso_name": "ubuntu-16.04-server-amd64.iso",
```
Comment (oinume) [inline ubuntu1604-desktop.json]: I think this should be `ubuntu-16.04.1-desktop-amd64.iso`, right?

Comment (jerryfromearth) [inline ubuntu1604-desktop.json]: @oinume , does `ubuntu-16.04.1-desktop-amd64.iso` really work for you?
There has been an automation(boot_command) problem with the desktop iso, so server iso was used instead(ubuntu-desktop is installed via provision scripts later). 
More discussion: https://github.com/boxcutter/ubuntu/issues/63

Comment (oinume) [inline ubuntu1604-desktop.json]: No, it didn't work. I understand the problem and I dropped my commit.

**Pull request merged**